include/Song.h

```diff
@@ -340,6 +340,7 @@ private slots:
 	QString m_fileName;
 	QString m_oldFileName;
 	bool m_modified;
+	bool m_loadOnLaunch;
 
```

Comment:
Yes,`song::loadProject( ... )` needs to know if the project is loaded on launch or from the gui. If there is a way to do this already we can drop this bool or if you have a better name for `m_loadOnLaunch` no one is happier than me.